backend/capellacollab/projects/toolmodels/modelsources/git/github/handler.py

```diff
@@ -69,16 +69,14 @@ def __get_file_from_repository(
         )
 
     async def get_file_from_repository(
-        self,
-        trusted_file_path: str,
-    ) -> bytes:
+        self, project_id: str, trusted_file_path: str
+    ) -> tuple[requests.Response, bytes]:
```

Comment:
As discussed either adjust the type hint to match the case that bytes can also be None or change it back to just `bytes` and raise an error if no file found